riva/proto/riva_nmt.proto

```diff
@@ -144,6 +144,8 @@ message TranslateTextRequest {
   // words not be translated should be specified as "<word>##".
   repeated string dnt_phrases = 5;
 
+  //Max Length variation controls the allowed difference between source and translated text.
```

Comment:
please confirm if below is correct. We can modify the comment as below for clarity.

`  // controls the maximum variation between the length of source and translated text in terms of words.
`